docs/config.json

```diff
@@ -851,6 +855,10 @@
       "last_report": "April 12th, 2021"
     },
     "cloud": {
+      "version": {
+        "control_plane": "8.3.4",
+        "agent": "8.3.4"
```

Comment:
@pierrebeaucamp I actually don't know the version of the Cloud Control Plane at the moment. What is the current version and keep in mind, when you do a deploy, you'll need to submit a PR to the docs repo to update these values as well.